

<!DOCTYPE html>
<html class="writer-html5" lang="en" data-content_root="../">
<head>
  <meta charset="utf-8" />
  <meta name="readthedocs-addons-api-version" content="1"><meta name="viewport" content="width=device-width, initial-scale=1" />

  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>About 3D Slicer &mdash; 3D Slicer  documentation</title>
      <link rel="stylesheet" type="text/css" href="../_static/pygments.css?v=03e43079" />
      <link rel="stylesheet" type="text/css" href="../_static/css/theme.css?v=e59714d7" />
      <link rel="stylesheet" type="text/css" href="../_static/css/custom.css?v=0defd4e4" />

  
      <script src="../_static/jquery.js?v=5d32c60e"></script>
      <script src="../_static/_sphinx_javascript_frameworks_compat.js?v=2cd50e6c"></script>
      <script src="../_static/documentation_options.js?v=5929fcd5"></script>
      <script src="../_static/doctools.js?v=9bcbadda"></script>
      <script src="../_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="../_static/js/theme.js"></script>
    <script src="../_static/js/versions.js"></script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="Getting Started" href="getting_started.html" />
    <link rel="prev" title="Welcome to 3D Slicer’s documentation!" href="../index.html" /> 
<script async type="text/javascript" src="/_/static/javascript/readthedocs-addons.js"></script><meta name="readthedocs-project-slug" content="slicer" /><meta name="readthedocs-version-slug" content="latest" /><meta name="readthedocs-resolver-filename" content="/user_guide/about.html" /><meta name="readthedocs-http-status" content="200" /></head>

<body class="wy-body-for-nav"> 
  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >

          
          
          <a href="../index.html" class="icon icon-home">
            3D Slicer
              <img src="../_static/3D-Slicer-Mark.png" class="logo" alt="Logo"/>
          </a>
              <div class="switch-menus">
                <div class="version-switch"></div>
                <div class="language-switch"></div>
              </div>
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>
        </div><div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <ul class="current">
<li class="toctree-l1 current"><a class="current reference internal" href="#">About 3D Slicer</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#what-is-3d-slicer">What is 3D Slicer?</a></li>
<li class="toctree-l2"><a class="reference internal" href="#license">License</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#historical-notes-about-the-license">Historical notes about the license</a></li>
<li class="toctree-l3"><a class="reference internal" href="#license-terms-and-reasons">License terms and reasons</a></li>
<li class="toctree-l3"><a class="reference internal" href="#status-compared-to-other-open-source-licenses">Status compared to other open source licenses</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#how-to-cite">How to cite</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#d-slicer-as-a-platform">3D Slicer as a platform</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#the-3d-slicer-name-and-logo">The 3D Slicer name and logo</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#individual-modules">Individual modules</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#acknowledgments">Acknowledgments</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#major-contributors">Major Contributors</a></li>
<li class="toctree-l3"><a class="reference internal" href="#groups-contributing-to-the-core-engineering-of-slicer-in-a-major-way">Groups Contributing to the Core Engineering of Slicer in a Major Way</a></li>
<li class="toctree-l3"><a class="reference internal" href="#funding-sources">Funding Sources</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#commercial-use">Commercial Use</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#slicer-s-license-makes-commercial-use-available">Slicer’s License makes Commercial Use Available</a></li>
<li class="toctree-l3"><a class="reference internal" href="#commercial-partners">Commercial Partners</a></li>
<li class="toctree-l3"><a class="reference internal" href="#d-slicer-based-products">3D Slicer based products</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#governance">Governance</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#code-of-conduct">Code of Conduct</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#our-pledge">Our Pledge</a></li>
<li class="toctree-l4"><a class="reference internal" href="#our-standards">Our Standards</a></li>
<li class="toctree-l4"><a class="reference internal" href="#enforcement-responsibilities">Enforcement Responsibilities</a></li>
<li class="toctree-l4"><a class="reference internal" href="#scope">Scope</a></li>
<li class="toctree-l4"><a class="reference internal" href="#enforcement">Enforcement</a></li>
<li class="toctree-l4"><a class="reference internal" href="#attribution">Attribution</a></li>
<li class="toctree-l4"><a class="reference internal" href="#deviations">Deviations</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="#decision-making-process">Decision-making process</a></li>
<li class="toctree-l3"><a class="reference internal" href="#benevolent-dictators-for-life">Benevolent dictators for life</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#contact-us">Contact us</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="getting_started.html">Getting Started</a></li>
<li class="toctree-l1"><a class="reference internal" href="get_help.html">Get Help</a></li>
<li class="toctree-l1"><a class="reference internal" href="user_interface.html">User Interface</a></li>
<li class="toctree-l1"><a class="reference internal" href="coordinate_systems.html">Coordinate systems</a></li>
<li class="toctree-l1"><a class="reference internal" href="data_loading_and_saving.html">Data Loading and Saving</a></li>
<li class="toctree-l1"><a class="reference internal" href="image_segmentation.html">Image Segmentation</a></li>
<li class="toctree-l1"><a class="reference internal" href="registration.html">Registration</a></li>
<li class="toctree-l1"><a class="reference internal" href="modules/index.html">Modules</a></li>
<li class="toctree-l1"><a class="reference internal" href="extensions.html">Extensions</a></li>
<li class="toctree-l1"><a class="reference internal" href="settings.html">Application settings</a></li>
<li class="toctree-l1"><a class="reference internal" href="../developer_guide/index.html">Developer Guide</a></li>
</ul>

        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap"><nav class="wy-nav-top" aria-label="Mobile navigation menu" >
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../index.html">3D Slicer</a>
      </nav>

      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="Page navigation">
  <ul class="wy-breadcrumbs">
      <li><a href="../index.html" class="icon icon-home" aria-label="Home"></a></li>
      <li class="breadcrumb-item active">About 3D Slicer</li>
      <li class="wy-breadcrumbs-aside">
              <a href="https://github.com/slicer/slicer/blob/main/Docs/user_guide/about.md" class="fa fa-github"> Edit on GitHub</a>
      </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
             
  <section class="tex2jax_ignore mathjax_ignore" id="about-3d-slicer">
<h1>About 3D Slicer<a class="headerlink" href="#about-3d-slicer" title="Link to this heading"></a></h1>
<section id="what-is-3d-slicer">
<h2>What is 3D Slicer?<a class="headerlink" href="#what-is-3d-slicer" title="Link to this heading"></a></h2>
<ul class="simple">
<li><p>A software application for visualization and analysis of medical image computing data sets. All commonly used data sets are supported, such as images, segmentations, surfaces, annotations, transformations, etc., in 2D, 3D, and 4D. Visualization is available on desktop and in virtual reality. Analysis includes segmentation, registration, and various quantifications.</p></li>
<li><p>A research software platform, which allows researchers to quickly develop and evaluate new methods and distribute them to clinical users. All features are available and extensible in Python and C++. A full Python environment is provided where any Python packages can be installed and combined with built-in features. Slicer has a built-in Python console and can act as a Jupyter notebook kernel with remote 3D rendering capabilities.</p></li>
<li><p>Product development platform, which allows companies to quickly prototype and release products to users. Developers can focus on developing new methods and do not need to spend time with redeveloping basic data import/export, visualization, interaction features. The application is designed to be highly customizable (with custom branding, simplified user interface, etc.). 3D Slicer is completely free and there are no restrictions on how it is used - it is up to the software distributor to ensure that the developed application is suitable for the intended use.</p></li>
</ul>
<p><strong>Note:</strong> There is no restriction on use, but Slicer is <strong>NOT</strong> approved for clinical use and the distributed application is intended for research use. Permissions and compliance with applicable rules are the responsibility of the user. For details on the license see <a class="reference external" href="https://www.slicer.org/wiki/License">here</a>.</p>
<p>Highlights:</p>
<ul class="simple">
<li><p>Free, <a class="reference external" href="https://en.wikipedia.org/wiki/Open_source">open-source</a> software available on multiple operating systems: Linux, macOS and Windows.</p></li>
<li><p>Multi organ: from head to toe.</p></li>
<li><p>Support for multi-modality imaging including, MRI, CT, US, nuclear medicine, and microscopy.</p></li>
<li><p>Real-time interface for medical devices, such as surgical navigation systems, imaging systems, robotic devices, and sensors.</p></li>
<li><p>Highly extensible: users can easily add more capabilities by installing additional modules from the Extensions manager, running custom Python scripts in the built-in Python console, run any executables from the application’s user interface, or implement custom modules in Python or C++.</p></li>
<li><p>Large and active user community.</p></li>
</ul>
</section>
<section id="license">
<h2>License<a class="headerlink" href="#license" title="Link to this heading"></a></h2>
<p>The 3D Slicer software is distributed under a BSD-style open source license that is broadly compatible with the Open Source Definition by <a class="reference external" href="https://opensource.org/">The Open Source Initiative</a> and contains no restrictions on legal uses of the software.</p>
<p>To use Slicer, please read the <a class="reference external" href="https://github.com/Slicer/Slicer/blob/main/License.txt">3D Slicer Software License Agreement</a> before downloading any binary releases of the Slicer.</p>
<section id="historical-notes-about-the-license">
<h3>Historical notes about the license<a class="headerlink" href="#historical-notes-about-the-license" title="Link to this heading"></a></h3>
<p>The Slicer License was drafted in 2005 by lawyers working for Brigham and Women’s Hospital (BWH), a teaching affiliate of Harvard Medical School, to be BSD-like but with a few extra provisions related to medical software.  It is specific to BWH so it’s not directly reusable, but it could serve as a template for projects with similar goals.</p>
<p>It was written in part because BWH was the prime contractor on an NIH-funded development consortium (<a class="reference external" href="https://www.na-mic.org/">NA-MIC</a>) and wanted all code contributions to be compatible with ultimate use in real-world medical products (that is, commercial FDA-approved medical devices, which are almost universally closed source even if they build on open software).  Compliance with the Slicer License was required for subcontractors, a group that included GE Research, Kitware and several universities (MIT, UNC…) who all reviewed and accepted this license.</p>
<p>The license has been in continuous use since 2005 for the 3D Slicer software package (<a class="reference external" href="http://slicer.org">slicer.org</a>) that as of 2021 has been downloaded more than a million times and has been referenced in about 12,000 academic publications (<a class="reference external" href="https://www.slicer.org/wiki/Main_Page/SlicerCommunity">https://www.slicer.org/wiki/Main_Page/SlicerCommunity</a>). Some of the code is also now being used in several medical products for which this license has been reviewed and accepted by the companies involved.</p>
</section>
<section id="license-terms-and-reasons">
<h3>License terms and reasons<a class="headerlink" href="#license-terms-and-reasons" title="Link to this heading"></a></h3>
<p>Here are some of the key points that BWH included in addition to BSD terms to make the license suit the case of a large hospital distributing open source medical software.</p>
<p>For using and redistributing 3D Slicer:</p>
<ul class="simple">
<li><p>The license states that the code is “designed for research” and “CLINICAL APPLICATIONS ARE NEITHER RECOMMENDED NOR ADVISED” to make it extra clear that any commercial clinical uses of the code are solely the responsibility of the user and not BWH or the other developers.  This is a disclaimer rather than a legal restriction.</p></li>
</ul>
<p>For making changes or adding any source code or data to 3D Slicer:</p>
<ul class="simple">
<li><p>Contributors explicitly grant royalty free rights if they contribute code covered by a patent they control (i.e. to avoid submarine patents).</p></li>
<li><p>No GPL or other copyleft code is allowed because that could make it complicated and risky to mix Slicer code with private intellectual property, which is often present in regulated medical products.</p></li>
<li><p>Contributors affirm that they have de-identified any patient data they contribute to avoid issues with HIPAA or related regulations.</p></li>
</ul>
</section>
<section id="status-compared-to-other-open-source-licenses">
<h3>Status compared to other open source licenses<a class="headerlink" href="#status-compared-to-other-open-source-licenses" title="Link to this heading"></a></h3>
<p>As of June 2021, the Slicer License has been used for over 15 years without incident. In May of 2021, a discourse user <a class="reference external" href="https://discourse.slicer.org/t/apply-for-osi-open-source-license-status/17791">suggested</a> submitting the license to <a class="reference external" href="https://opensource.org/approval">the OSI license review process</a>. After some discussion and hearing no objections, the community leadership decided to <a class="reference external" href="https://lists.opensource.org/pipermail/license-review_lists.opensource.org/2021-May/thread.html">submit the license for review</a>. Although the OSI process is not legally binding, the discussion could give potential Slicer users perspective on how provisions of the license compare with other commonly used licenses.
The discussion concluded that bundling the contribution agreement in the license makes it non-approvable by OSI and the requirement to use the software for legal purposes may not be consistent with the <a class="reference external" href="https://opensource.org/osd">Open Source Definition</a>. Otherwise the license terms appear not to be controversial. Interested parties should review the <a class="reference external" href="https://lists.opensource.org/pipermail/license-review_lists.opensource.org/2021-June/thread.html#5166">full discussion</a> for details.</p>
</section>
</section>
<section id="how-to-cite">
<h2>How to cite<a class="headerlink" href="#how-to-cite" title="Link to this heading"></a></h2>
<section id="d-slicer-as-a-platform">
<h3>3D Slicer as a platform<a class="headerlink" href="#d-slicer-as-a-platform" title="Link to this heading"></a></h3>
<p>To acknowledge 3D Slicer as a platform, please cite the <a class="reference external" href="https://www.slicer.org/">Slicer web site</a> and the following publications when publishing work that uses or incorporates 3D Slicer:</p>
<p><strong>Fedorov A., Beichel R., Kalpathy-Cramer J., Finet J., Fillion-Robin J-C., Pujol S., Bauer C., Jennings D., Fennessy F.M., Sonka M., Buatti J., Aylward S.R., Miller J.V., Pieper S., Kikinis R. <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3466397/pdf/nihms383480.pdf">3D Slicer as an Image Computing Platform for the Quantitative Imaging Network</a>. Magnetic Resonance Imaging. 2012 Nov;30(9):1323-41. PMID: 22770690. PMCID: PMC3466397.</strong></p>
</section>
</section>
<section id="the-3d-slicer-name-and-logo">
<h2>The 3D Slicer name and logo<a class="headerlink" href="#the-3d-slicer-name-and-logo" title="Link to this heading"></a></h2>
<p>3D Slicer and the logo are trademarks of Brigham and Women’s Hospital (BWH) and may not be used without permission.  Such permission is broadly granted for academic or commercial uses, such as documenting the use of Slicer in your project or promoting the use of Slicer by others.  Please use the original Slicer logo colors and do not alter the shape or text.  Using Slicer to imply that BWH or the Slicer community endorses your product or project is not permitted without permission.  For other uses please contact Ron Kikinis (<a class="reference external" href="mailto:kikinis&#37;&#52;&#48;bwh&#46;harvard&#46;edu">kikinis<span>&#64;</span>bwh<span>&#46;</span>harvard<span>&#46;</span>edu</a>) and Steve Pieper (<a class="reference external" href="mailto:pieper&#37;&#52;&#48;bwh&#46;harvard&#46;edu">pieper<span>&#64;</span>bwh<span>&#46;</span>harvard<span>&#46;</span>edu</a>).</p>
<section id="individual-modules">
<h3>Individual modules<a class="headerlink" href="#individual-modules" title="Link to this heading"></a></h3>
<p>To acknowledge individual modules: each module has an acknowledgment tab in the top section. Information about contributors and funding source can be found there:</p>
<p><img alt="" src="https://github.com/Slicer/Slicer/releases/download/docs-resources/acknowledgement_ack_tab.png" /></p>
<p>Additional information (including information about the underlying publications) can be typically found on the manual pages accessible through the help tab in the top section:</p>
<p><img alt="" src="https://github.com/Slicer/Slicer/releases/download/docs-resources/acknowledgement_help_tab.png" /></p>
</section>
</section>
<section id="acknowledgments">
<h2>Acknowledgments<a class="headerlink" href="#acknowledgments" title="Link to this heading"></a></h2>
<p>Slicer is made possible through contributions from an international community of scientists from a multitude of fields, including engineering and biomedicine. The following sections give credit to some of the major contributors to the 3D Slicer core effort. Each 3D Slicer extension has a separate acknowledgements page with information specific to that extension.</p>
<p>Ongoing Slicer support depends on YOU</p>
<p>Please give the Slicer repository a star on github. This is an easy way to show thanks and it can help us qualify for useful services that are only open to widely recognized open projects.
Don’t forget to cite our publications because that helps us get new grant funding.
If you find Slicer is helpful like the community please get involved. You don’t need to be a programmer to help!</p>
<section id="major-contributors">
<h3>Major Contributors<a class="headerlink" href="#major-contributors" title="Link to this heading"></a></h3>
<ul class="simple">
<li><p>Ron Kikinis: Principal Investigator</p></li>
<li><p>Steve Pieper: Chief Architect</p></li>
<li><p>Jean-Christophe Fillion-Robin: Lead Developer</p></li>
<li><p>Nicole Aucoin</p></li>
<li><p>Stephen Aylward</p></li>
<li><p>Andrey Fedorov</p></li>
<li><p>Noby Hata</p></li>
<li><p>Hans Johnson</p></li>
<li><p>Tina Kapur</p></li>
<li><p>Gabor Fichtinger</p></li>
<li><p>Andras Lasso</p></li>
<li><p>Csaba Pinter</p></li>
<li><p>Jim Miller</p></li>
<li><p>Sonia Pujol: Director of Training</p></li>
<li><p>Junichi Tokuda</p></li>
<li><p>Lauren O’Donnell</p></li>
<li><p>Andinet Enquobahrie</p></li>
<li><p>Beatriz Paniagua</p></li>
</ul>
<p><em>Contributors are not only developers, but also individual helping to secure funding and move the platform forward.</em></p>
</section>
<section id="groups-contributing-to-the-core-engineering-of-slicer-in-a-major-way">
<h3>Groups Contributing to the Core Engineering of Slicer in a Major Way<a class="headerlink" href="#groups-contributing-to-the-core-engineering-of-slicer-in-a-major-way" title="Link to this heading"></a></h3>
<ul class="simple">
<li><p>SPL: Ron Kikinis, Nicole Aucoin, Lauren O’Donnell, Andrey Fedorov, Isaiah Norton, Sonia Pujol, Noby Hata, Junichi Tokuda</p></li>
<li><p>Isomics: Steve Pieper, Alex Yarmarkovich</p></li>
<li><p>Kitware: Jean-Christophe Fillion-Robin, Julien Finet, Will Schroeder, Stephen Aylward, Andinet Enquobahrie, Beatriz Paniagua, Matt McCormick, Johan Andruejol, Max Smolens, Alexis Girault, Sam Horvath</p></li>
<li><p>University of Iowa: Hans Johnson</p></li>
<li><p>GE: Jim Miller</p></li>
<li><p>Perk Lab, Queen’s University: Andras Lasso, Tamas Ungi, Csaba Pinter, Gabor Fichtinger</p></li>
<li><p>Kapteyn Astronomical Institute, University of Groningen: Davide Punzo</p></li>
</ul>
</section>
<section id="funding-sources">
<h3>Funding Sources<a class="headerlink" href="#funding-sources" title="Link to this heading"></a></h3>
<p>Many of the activities around the Slicer effort are made possible through funding from public and private sources. The National Institutes of Health of the USA is a major contributor through a variety of competitive grants and contracts. Funding sources that contribute to development of Slicer core or extensions include (ordered by End Date):</p>
<table border="1" class="docutils">
<thead>
<tr>
<th>Grant Number</th>
<th>Description</th>
<th>Grant PIs</th>
<th>Start Date</th>
<th>End Date</th>
<th>Title</th>
</tr>
</thead>
<tbody>
<tr>
<td>CHOP</td>
<td>CHOP Frontier Program</td>
<td>Matthew Jolley</td>
<td>2024-07-01</td>
<td>to date</td>
<td>Children’s Hospital of Philadelphia Frontier Programs conduct visionary research that translates to cutting-edge care]</td>
</tr>
<tr>
<td>NSF/DBI 2301405</td>
<td>MorphoCloud and MorphoDepot</td>
<td>Murat Maga</td>
<td>2024-03-01</td>
<td>2028-02-28</td>
<td>MorphoCloud: A Cloud Powered, Open-Source Platform For Research, Teaching And Collaboration In 3d Digital Morphology And Beyond</td>
</tr>
<tr>
<td>NSF/OAC 2118240</td>
<td>Photogrammetry extension</td>
<td>Murat Maga</td>
<td>2021-09-15</td>
<td>2026-08-31</td>
<td>Imageomics: A New Frontier of Biological Information Powered by Knowledge-Guided Machine Learning</td>
</tr>
<tr>
<td>AV 993932</td>
<td>SlicerHeart for Single Ventricle</td>
<td>Matthew Jolley</td>
<td>2023-03-01</td>
<td>2026-02-28</td>
<td>Computational Modelling of the Atrioventricular Valve Repair Single Ventricle Patients with Atrioventricular Canal]</td>
</tr>
<tr>
<td>NIH P01HD104435</td>
<td>Mouse Embryo Multi-Organ Segmentations (MEMOS), ANTsPy Extension, Terminologies</td>
<td>Murat Maga</td>
<td>2021-01-11</td>
<td>2025-12-31</td>
<td>3D Quantitative Analysis of Mouse Models of Structural Birth Defects Through Computational Anatomy</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=10029738&amp;icde=51006191">NIH 1R01HL153166-01</a></td>
<td>Computer Modeling of the Tricuspid Valve in HLHS</td>
<td>Matthew Jolley</td>
<td>2021-06-30</td>
<td>2025-06-30</td>
<td>Computer Modeling of the Tricuspid Valve in Hypoplastic Left Heart Syndrome</td>
</tr>
<tr>
<td>NIH R03OD032627</td>
<td>Dense Correspondence Analysis Toolkit (DeCA)</td>
<td>Murat Maga</td>
<td>2021-09-22</td>
<td>2023-09-21</td>
<td>Deep Phenotyping of 3D Data for Candidate Gene Selection from Kids First Studies</td>
</tr>
<tr>
<td><a href="https://chanzuckerberg.com/eoss/proposals/3d-slicer-in-my-language-internationalization-and-usability-improvements/">CZI EOSS 4</a></td>
<td>Slicer internationalization and usability</td>
<td>Pujol, Sonia</td>
<td>2021-09-01</td>
<td>2023-09-01</td>
<td>3D Slicer in My Language: Internationalization and Usability Improvements</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=9115586&amp;icde=31485398&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=3&amp;csb=default&amp;cs=ASC">NIH 4P41EB015902</a></td>
<td>Neuroimage Analysis Center</td>
<td>Ron Kikinis</td>
<td>2013-08-01</td>
<td>2023-08-31</td>
<td>Application of Slicer to image-guided neurosurgery and other applications through steered computatiton and image navigation databases</td>
</tr>
<tr>
<td>CANARIE RS3-036</td>
<td>SlicerAIGT</td>
<td>Fichtinger, Gabor</td>
<td>2020-10-01</td>
<td>2023-03-31</td>
<td>Develop free open source research software for AI-powered image guided therapy on Slicer platform</td>
</tr>
<tr>
<td><a href="https://reporter.nih.gov/search/-oPV-FDakki3bPslkjMUwg/project-details/10053340">NIH R01MH112748</a></td>
<td>High-accuracy brain segmentation tools</td>
<td>Bouix, Sylvain</td>
<td>2017-12-01</td>
<td>2022-10-31</td>
<td>High Resolution, Comprehensive Atlases of the Human Brain Morphology</td>
</tr>
<tr>
<td>NSF/DBI 1759883</td>
<td>SlicerMorph extension, Markups infrastructure</td>
<td>Murat Maga</td>
<td>2018-08-01</td>
<td>2022-07-31</td>
<td>An Integrated Platform for Retrieval, Visualization and Analysis of 3D Morphology From Digital Biological Collections</td>
</tr>
<tr>
<td><a href="https://www.sbir.gov/sbirsearch/detail/1683213">NIH R44DK115332</a></td>
<td>Renal Biopsy</td>
<td>Enquobahrie, Andinet A.</td>
<td>2019-08-13</td>
<td>2021-07-31</td>
<td>Advanced virtual simulator for real-time ultrasound-guided renal biopsy training</td>
</tr>
<tr>
<td>ICEX 0202101723</td>
<td>SlicerVR Collaboration</td>
<td>Juan Ruiz Alzola</td>
<td>2021-01-16</td>
<td>2021-12-31</td>
<td>Sistema Virtual Colaborativa Aplicación Médicas</td>
</tr>
<tr>
<td><a href="https://govtribe.com/award/federal-grant-award/project-grant-r01eb025212">NIH R01EB025212</a></td>
<td>Software annotation</td>
<td>Enquobahrie, Andinet A.</td>
<td>2019-07-02</td>
<td>2021-03-31</td>
<td>Software for Practical Annotation and Exchange of Virtual Anatomy</td>
</tr>
<tr>
<td>CANARIE RS319</td>
<td>SlicerIGT</td>
<td>Fichtinger, Gabor</td>
<td>2019-10-01</td>
<td>2020-09-30</td>
<td>Canadian Research Software Infrastructure</td>
</tr>
<tr>
<td>CANARIE RS214</td>
<td>SlicerRT</td>
<td>Fichtinger, Gabor</td>
<td>2017-07-01</td>
<td>2020-09-30</td>
<td>Canadian Research Software Infrastructure</td>
</tr>
<tr>
<td>CHOP</td>
<td>SlicerHeart</td>
<td>Matthew Jolley</td>
<td>2015-08-15</td>
<td>2020-08-15</td>
<td>Pediatric cardiac valve modeling, Children Hospital of Philadelphia</td>
</tr>
<tr>
<td><a href="https://mt4sd.ulpgc.es/en/macbioidi-project/">MAC/1.1b/098</a></td>
<td>MACBIOLDI</td>
<td>Juan Ruiz Alzola</td>
<td>2017-01-01</td>
<td>2020-09-30</td>
<td>Medical Technology for Sustainable Development (MedTec4SusDev)</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_details.cfm?aid=8855115&amp;icde=27026518">NIH 2P41EB015898</a></td>
<td>DiffusionMRI</td>
<td>Tempany, Clare M</td>
<td>2004-04-01</td>
<td>2020-06-30</td>
<td>Image Guided Therapy Center</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=9125821&amp;icde=31485478&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=2&amp;csb=default&amp;cs=ASC">NIH 5P41EB015898</a></td>
<td>National Center for Image Guided Therapy</td>
<td>Clare Tempany</td>
<td>2004-04-01</td>
<td>2020-06-30</td>
<td>Use of Slicer in a wide array of image-guided therapy research for prostate cancer, neurosurgery, and image naviation</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=9123966&amp;icde=31459429&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=2&amp;csb=default&amp;cs=ASC">NIH 1R01EB021391</a></td>
<td>Shape</td>
<td>Paniagua, Beatriz</td>
<td>2016-09-19</td>
<td>2020-06-30</td>
<td>SHAPE ANALYSIS TOOLBOX FOR MEDICAL IMAGE COMPUTING PROJECTS</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8875289&amp;icde=27050248">NIH U24CA194354</a></td>
<td>Slicer-Radiomics-U24</td>
<td>Aerts, Hugo</td>
<td>2015-04-01</td>
<td>2020-03-31</td>
<td>Quantitative Radiomics System Decoding the Tumor Phenotype</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8799943&amp;icde=27026470">NIH 1U01CA190234</a></td>
<td>Slicer-Radiomics-U01</td>
<td>Aerts, Hugo</td>
<td>2015-01-01</td>
<td>2019-12-01</td>
<td>Genotype And Imaging Phenotype Biomarkers In Lung Cancer</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_details.cfm?aid=9127923&amp;icde=31460433&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=2&amp;csb=default&amp;cs=ASC">NIH 5U24CA180924</a></td>
<td>Spatially mapped molecular data analysis</td>
<td>Joel Saltz</td>
<td>2014-09-01</td>
<td>2019-08-31</td>
<td>Tools to Analyze Morphology and Spatially Mapped Molecular Data</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8828624&amp;icde=27036841&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASC">NIH 5R01CA184354</a></td>
<td>NIRFAST (Dartmouth)</td>
<td>Davis, Scott C.</td>
<td>2014-04-01</td>
<td>2019-02-28</td>
<td>Mri Fluorescence Tomography For Quantifying Tumor Receptor Concentration In Vivo</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=9465772&amp;icde=36620728&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=2&amp;csb=default&amp;cs=ASC&amp;pball=">NIH R43DE027595</a></td>
<td>VROrthognathic</td>
<td>Paniagua, Beatriz</td>
<td>2017-09-07</td>
<td>2018-09-06</td>
<td>High-Fidelity Virtual Reality Trainer for Orthognathic Surgery</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=9111256&amp;icde=31459429&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASC">NIH 1R21DE025306</a></td>
<td>CMF</td>
<td>Paniagua, Beatriz</td>
<td>2016-09-01</td>
<td>2018-08-31</td>
<td>TEXTURAL BIOMARKERS OF ARTHRITIS FOR THE SUBCHONDRAL BONE IN THE TEMPOROMANDIBULAR JOINT</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8462842&amp;icde=27164806&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASC">NIH 1U01NS082086</a></td>
<td>HD_SHAPEANALSS</td>
<td>Gerig, Guido</td>
<td>2012-09-28</td>
<td>2018-08-31</td>
<td>4D Shape Analysis for Modeling Spatiotemporal Change Trajectories in Huntington’s</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8911287&amp;icde=27026906">NIH U24 CA180918</a></td>
<td><a href="https://qiicr.org">QIICR</a></td>
<td>Ron Kikinis, Andrey Fedorov</td>
<td>2013-09-04</td>
<td>2018-08-31</td>
<td>Quantitative Image Informatics for Cancer Research (QIICR)</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8900362&amp;icde=27164764&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASC">NIH 5R01NS055903</a></td>
<td>HD_KIDS</td>
<td>Nopoulos, Peggy</td>
<td>2009-03-01</td>
<td>2018-07-31</td>
<td>Growth and Development of the Striatum in Huntington's Disease</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8971083&amp;icde=27026834">NIH 1U01CA199459</a></td>
<td>SlicerDMRI Diffusion MRI</td>
<td>O'Donnell, Lauren Jean</td>
<td>2015-09-22</td>
<td>2018-07-31</td>
<td>Open Source Diffusion MRI Technology For Brain Cancer Research</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=9100712&amp;icde=34329960">NIH 5R01EB020667-02</a></td>
<td>OpenIGTLink</td>
<td>Tokuda, Junichi</td>
<td>2015-07-01</td>
<td>2018-06-30</td>
<td>OpenIGTLink: A Network Communication Interface for Closed-Loop Image-Guided Interventions</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8890837&amp;icde=27036647&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASC">NIH 5P41EB015902</a></td>
<td>DiffusionMRI</td>
<td>Kikinis, Ron</td>
<td>2013-08-01</td>
<td>2018-05-31</td>
<td>Neuroimaging Analysis Center (Nac)</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=9141675&amp;icde=31459353&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=5&amp;csb=default&amp;cs=ASC">NIH 2R42HD081712</a></td>
<td>Craniosynostosis</td>
<td>Linguraru, Marius George</td>
<td>2016-05-01</td>
<td>2018-04-30</td>
<td>IMAGE-GUIDED PLANNING SYSTEM FOR SKULL CORRECTION IN CHILDREN WITH CRANIOSYNOSTOSIS: PHASE II</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8817256&amp;icde=27036729&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASC">NIH R01CA160902</a></td>
<td>DWI</td>
<td>Maier, Stephan E</td>
<td>2012-04-01</td>
<td>2018-02-28</td>
<td>Advancement And Validation Of Prostate Diffusion And Spectroscopic Mri</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8576556&amp;icde=18353487">NIH 1R01DE024450</a></td>
<td>CMF</td>
<td>Cevidanes, Lucia</td>
<td>2013-09-10</td>
<td>2017-08-31</td>
<td>Quantification Of 3D Bony Changes In Temporomandibular Joint Osteoarthritis</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8979242&amp;icde=27036988&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASC">NIH 2R42CA167907</a></td>
<td>PET/CT Calibration Phantom</td>
<td>Kinahan, Paul E</td>
<td>2012-05-01</td>
<td>2017-07-31</td>
<td>Calibrated Methods For Quantitative Pet/Ct Imaging Phase Ii</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8979242&amp;icde=27036988&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASC">NIH R42CA167907</a></td>
<td>PET/CT Calibration Phantom</td>
<td>Kinahan, Paul E.</td>
<td>2012-05-01</td>
<td>2017-07-01</td>
<td>Calibrated Methods For Quantitative Pet/Ct Imaging Phase Ii</td>
</tr>
<tr>
<td>NA</td>
<td>HD_TRACKON</td>
<td>Tabrizi, Sarah</td>
<td>2012-01-01</td>
<td>2016-12-31</td>
<td>TRACK-ON HD</td>
</tr>
<tr>
<td>CCO ACRU</td>
<td>SlicerRT</td>
<td>Fichtinger, Gabor</td>
<td>2011-01-01</td>
<td>2016-12-31</td>
<td>Cancer Care Ontario Applied Cancer Research Unit, Canada</td>
</tr>
<tr>
<td>CCO OCAIRO</td>
<td>SlicerRT</td>
<td>Jaffray, David</td>
<td>2011-01-01</td>
<td>2016-12-31</td>
<td>Ontario Consortium for Adaptive Interventions in Radiation Oncology, Canada</td>
</tr>
<tr>
<td>NA</td>
<td>HD_TRAJECTORY</td>
<td>Kim, Eun Young</td>
<td>2014-11-01</td>
<td>2016-10-31</td>
<td>Developing a Robust Segmentation Pipeline That Allows for Consistent Trajectory Estimation of HD Gene Positive Individuals Across Multiple Longitudinal MRI Sites</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8778815&amp;icde=27036063&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASCIMAGE-GUIDED">NIH 1R41HD081712</a></td>
<td>Craniosynostosis</td>
<td>Linguraru, Marius George</td>
<td>2014-09-26</td>
<td>2016-08-31</td>
<td>Image-Guided Planning System For Skull Correction In Children With Craniosynostos</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8338456&amp;icde=27164778&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=4&amp;csb=default&amp;cs=ASC">NIH 5R01NS040068</a></td>
<td>HD_PREDICT</td>
<td>Paulsen, Jane</td>
<td>2000-08-01</td>
<td>2016-08-31</td>
<td>Neurobiological Predictors of Huntington's Disease (PREDICT-HD)</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8863934&amp;icde=27037113&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASC">NIH 3R42CA153488</a></td>
<td>PET-CT guided needle biopsy</td>
<td>Cleary, Kevin R.</td>
<td>2012-09-01</td>
<td>2016-08-01</td>
<td>Improving Liver Lesion Biopsy In The Ct Suite Through Fusion With Pet Images</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8710950&amp;icde=27036891">NIH 1R43DE024334</a></td>
<td>OrthognathicTrac</td>
<td>Enquobahrie, Andinet A.</td>
<td>2014-08-05</td>
<td>2016-07-31</td>
<td>Real-Time Image Guidance For Improved Orthognathic Surgery</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8272742&amp;icde=13552329">NIH 1R01EB014947</a></td>
<td>PediatricRadiologicDecisionSupport</td>
<td>Murphy, Shawn N</td>
<td>2012-08-01</td>
<td>2016-07-31</td>
<td>Mi2B2 Enabled Pediatric Radiological Decision Support</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8906771&amp;icde=27026518">NIH 5R01CA111288 </a></td>
<td>ProstateBRP</td>
<td>Tempany, Clare M.</td>
<td>2004-12-01</td>
<td>2016-07-01</td>
<td>Enabling Technologies For Mri-Guided Prostate Interventions</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8707214&amp;icde=27026645">NIH 5U01CA151261</a></td>
<td>ProstateQIN</td>
<td>Fennessy, Fiona</td>
<td>2010-09-01</td>
<td>2016-07-01</td>
<td>Quantitative Mri Of Prostate Cancer As A Biomarker And Guide For Treatment</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8596213&amp;icde=27164895&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASC">NIH 1U01NS082074</a></td>
<td>HD_GENETICS</td>
<td>Calhoun/Turner</td>
<td>2013-07-01</td>
<td>2016-06-30</td>
<td>Imaging and Genetics in Huntington's Disease</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8529927&amp;icde=27164835&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASC">NIH 1U01NS083173</a></td>
<td>HD_PET</td>
<td>Feigin, Andrew</td>
<td>2013-07-01</td>
<td>2016-06-30</td>
<td>Brain Network Imaging: A Novel Biomarker for Preclinical Huntington’s Disease</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8384153&amp;icde=27037397&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASC">NIH 1R01CA170665</a></td>
<td><a href="https://tubetk.org">TubeTK</a></td>
<td>Dayton, Paul A.</td>
<td>2012-09-01</td>
<td>2016-06-01</td>
<td>Micro-Tumor Detection By Quantifying Tumor-Induced Vascular Abnormalities</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8652000&amp;icde=27164795&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASC">NIH 1U01NS083223</a></td>
<td>HD_WHITEMATTER</td>
<td>Westin, Carl-Fredrik</td>
<td>2014-01-01</td>
<td>2015-12-31</td>
<td>Characterization of White Matter in Huntington’s Disease Using Diffusion MRI</td>
</tr>
<tr>
<td>CCO RC</td>
<td>SlicerRT</td>
<td>Fichtinger, Gabor</td>
<td>2010-01-01</td>
<td>2015-12-31</td>
<td>Cancer Care Ontario Research Chair, Canada</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8462829&amp;icde=27164863&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASC">NIH 1U01NS082083</a></td>
<td>HD_FMRI_DWI</td>
<td>Rao, Stephen Mark</td>
<td>2012-09-26</td>
<td>2015-08-31</td>
<td>Functional Connectivity in Premanifest Huntington’s Disease</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8905274&amp;icde=27036277&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASC">NIH R41CA196565</a></td>
<td>Duke Prostate Registration</td>
<td>Palmeri, Mark L. &amp; McCormick, Matthew M.</td>
<td>2015-04-01</td>
<td>2015-04-01</td>
<td>Prostate Cancer Assessment Via Integrated 3D Arfi Elasticity Imaging And Multi-Parametric Mri</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8472102&amp;icde=27037328&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASC">NIH 1R43EB016621</a></td>
<td><a href="https://tubetk.org">TubeTK</a></td>
<td>Aylward, Stephen R.</td>
<td>2013-05-01</td>
<td>2015-04-01</td>
<td>In-Field Fast Procedure Support And Automation</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8453963&amp;icde=27037364&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASC">NIH 1R41NS081792</a></td>
<td><a href="https://tubetk.org">TubeTK</a></td>
<td>Aylward, Stephen R.</td>
<td>2013-01-01</td>
<td>2014-12-01</td>
<td>Multimodality Image-Based Assessment System For Traumatic Brain Injury</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8390856&amp;icde=27037039&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASC">NIH 2R42CA153488</a></td>
<td>PET-CT guided needle biopsy</td>
<td>Cleary, Kevin R.</td>
<td>2012-09-01</td>
<td>2014-08-01</td>
<td>Improving Liver Lesion Biopsy In The Ct Suite Through Fusion With Pet Images</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8252988&amp;icde=27037450&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASC">NIH 1R43CA165621</a></td>
<td><a href="https://tubetk.org">TubeTK</a></td>
<td>Aylward, Stephen R</td>
<td>2012-12-01</td>
<td>2014-08-01</td>
<td>Quantitative Ultrasound Analysis Of Vascular Morphology For Cancer Assessment</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8462830&amp;icde=27164813&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASC">NIH 5U01NS082085</a></td>
<td>HD_SUBCORTICAL_SHAPE</td>
<td>Miller, Michael &amp; Ross, Christopher</td>
<td>2012-09-26</td>
<td>2014-07-31</td>
<td>Basal Ganglia Shape Analysis and Circuitry in Huntington's Disease</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8501010&amp;icde=27164945&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=5&amp;csb=default&amp;cs=ASC">NIH 5U54EB005149</a></td>
<td>HD_DWI</td>
<td>Kikinis, Ron</td>
<td>2010-09-30</td>
<td>2014-06-30</td>
<td>National Alliance for Medical Image Computing</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8077226&amp;icde=27164732&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASC">NIH 5R01NS054893</a></td>
<td>HD_FMRI</td>
<td>Paulsen, Jane</td>
<td>2007-05-15</td>
<td>2013-04-30</td>
<td>Cognitive and Functional Brain Changes in Preclinical Huntington's Disease (HD)</td>
</tr>
<tr>
<td><a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=7999618&amp;icde=27037084&amp;ddparam=&amp;ddvalue=&amp;ddsub=&amp;cr=1&amp;csb=default&amp;cs=ASC">NIH R41CA153488</a></td>
<td>PET-CT guided needle biopsy</td>
<td>Cleary, Kevin R.</td>
<td>2010-07-01</td>
<td>2012-06-01</td>
<td>Improving Liver Lesion Biopsy In The Ct Suite Through Fusion With Pet Images</td>
</tr>
</tbody>
</table>
</section>
</section>
<section id="commercial-use">
<h2>Commercial Use<a class="headerlink" href="#commercial-use" title="Link to this heading"></a></h2>
<p>We invite commercial entities to use 3D Slicer.</p>
<section id="slicer-s-license-makes-commercial-use-available">
<h3>Slicer’s License makes Commercial Use Available<a class="headerlink" href="#slicer-s-license-makes-commercial-use-available" title="Link to this heading"></a></h3>
<ul class="simple">
<li><p>3D Slicer is a free open source software distributed under a BSD style license.</p></li>
<li><p>The license does not impose restrictions on the use of the software.</p></li>
<li><p>3D Slicer is NOT FDA approved. It is the users responsibility to ensure compliance with applicable rules and regulations.</p></li>
<li><p>For details, please see the 3D Slicer Software License Agreement.</p></li>
</ul>
</section>
<section id="commercial-partners">
<h3>Commercial Partners<a class="headerlink" href="#commercial-partners" title="Link to this heading"></a></h3>
<ul class="simple">
<li><p><a class="reference external" href="https://ebatinca.com/">Ebatinca SL</a> is an international technology company in Las Palmas, Spain focused on technology for sustainable development. Primary areas: ultrasound navigation and training, collaborative VR, research support, custom Slicer-based solutions.</p></li>
<li><p><a class="reference external" href="https://isomics.com/">Isomics</a> uses 3D Slicer in a variety of academic and commercial research partnerships in fields such as planning and guidance for neurosurgery, quantitative imaging for clinical trials, clinical image informatics.</p></li>
<li><p><a class="reference external" href="https://www.kitware.com/opensource/slicer.html">Kitware</a> focuses on solving the world’s most complex scientific challenges through customized software solutions. The company has a long history of contributing to open source platforms that serve as the foundation of many medical visualization and data processing applications. Kitware helps customers develop commercial products based on 3D Slicer and has used the platform to rapidly prototype solutions in nearly every aspect of medical imaging.</p></li>
</ul>
<p><em>Listed in alphabetical order.</em></p>
</section>
<section id="d-slicer-based-products">
<h3>3D Slicer based products<a class="headerlink" href="#d-slicer-based-products" title="Link to this heading"></a></h3>
<p>Many companies prefer not to disclose what software components they use in their products, therefore here we can only list a few commercial products that are based on 3D Slicer:</p>
<ul class="simple">
<li><p>Allen Institute for Brain Science: Allen Institute for Brain Science is developing Cell Locator, a Desktop application for manually aligning specimens to annotated 3D spaces. See more information on this <a class="reference external" href="https://blog.kitware.com/cell-locator-a-3d-slicer-based-desktop-application-that-manually-aligns-specimens-to-annotated-3d-spaces-developed-for-the-allen-institute-for-brain-science/">Kitware blog</a>.</p></li>
<li><p><a class="reference external" href="https://polarean.com/">Polarean, Inc.</a>: Polarean’s XENOVIEW VDP is an FDA-approved software built on 3D Slicer for visualization and evaluation of lung ventilation. See more information on this <a class="reference external" href="https://www.kitware.com/kitware-supports-development-and-launch-of-fda-cleared-mr-image-processingsoftware-for-lung-ventilation-imaging/">Kitware blog</a>.</p></li>
<li><p>Radiopharmaceutical Imaging and Dosimetry: RPTDose, a 3D Slicer-based application that streamlines and integrates quantitative imaging analysis and dose estimation techniques to guide and optimize the use of radiopharmaceutical therapy agents in clinical trials. See more information on this <a class="reference external" href="https://blog.kitware.com/kitware-customer-highlight-radiopharmaceutical-imaging-and-dosimetry-llc-rapid/">Kitware blog</a>.</p></li>
<li><p><a class="reference external" href="https://sonovol.com/">SonoVol</a> developed a whole-body ultrasound imaging system for small animals. This start-up company arose from research in the Department of Biomedical Engineering at the University of North Carolina at Chapel Hill. Their team is now part of Revvity, Inc. See more information on this <a class="reference external" href="https://blog.kitware.com/kitware-customer-highlight-sonovol/">Kitware blog</a> and their <a class="reference external" href="https://www.perkinelmer.com/category/ultrasound-imaging">product website</a>.</p></li>
<li><p>Xoran Technologies: Image-guided Platform for Deep Brain Stimulation Surgery. See more information on this <a class="reference external" href="https://blog.kitware.com/xoran-technologies-and-kitware-collaborate-on-image-guided-platform-for-deep-brain-stimulation-surgery/">Kitware blog</a>.</p></li>
<li><p><a class="reference external" href="https://www.xstrahl.com/">Xstrahl</a> is developing a Small Animal Radiation Research Platform (SARRP) that uses 3D Slicer as its front-end application for radiation therapy beam placement and system control. See more information on this <a class="reference external" href="https://blog.kitware.com/kitware-customer-highlight-muriplan-from-xstrahl-a-3d-slicer-based-radiotherapy-treatment-planning-system/">Kitware blog</a>.</p></li>
</ul>
<p><em>Listed in alphabetical order.</em></p>
</section>
</section>
<section id="governance">
<h2>Governance<a class="headerlink" href="#governance" title="Link to this heading"></a></h2>
<p>This section introduces the guiding principles and decision-making framework of the
Slicer project, including the Code of Conduct and governance model that describe how
decisions are made and who is responsible for maintaining the health of the project.</p>
<section id="code-of-conduct">
<h3>Code of Conduct<a class="headerlink" href="#code-of-conduct" title="Link to this heading"></a></h3>
<p>The Slicer community is committed to fostering a welcoming, respectful, and
inclusive environment. Details are outlined in the Code of Conduct below.</p>
<!--
  Notes: Includes CODE_OF_CONDUCT.md content, excluding its top-level header
  and intro.
-->
<section id="our-pledge">
<h4>Our Pledge<a class="headerlink" href="#our-pledge" title="Link to this heading"></a></h4>
<p>We as members, contributors, and leaders pledge to make participation in our
community a harassment-free experience for everyone, regardless of age, body
size, visible or invisible disability, ethnicity, sex characteristics, gender
identity and expression, level of experience, education, socio-economic status,
nationality, personal appearance, race, caste, color, religion, or sexual
identity and orientation.</p>
<p>We pledge to act and interact in ways that contribute to an open, welcoming,
diverse, inclusive, and healthy community.</p>
</section>
<section id="our-standards">
<h4>Our Standards<a class="headerlink" href="#our-standards" title="Link to this heading"></a></h4>
<p>Examples of behavior that contributes to a positive environment for our
community include:</p>
<ul class="simple">
<li><p>Demonstrating empathy and kindness toward other people</p></li>
<li><p>Being respectful of differing opinions, viewpoints, and experiences</p></li>
<li><p>Giving and gracefully accepting constructive feedback</p></li>
<li><p>Accepting responsibility and apologizing to those affected by our mistakes,
and learning from the experience</p></li>
<li><p>Focusing on what is best not just for us as individuals, but for the overall
community</p></li>
</ul>
<p>Examples of unacceptable behavior include:</p>
<ul class="simple">
<li><p>The use of sexualized language or imagery, and sexual attention or advances of
any kind</p></li>
<li><p>Trolling, insulting or derogatory comments, and personal or political attacks</p></li>
<li><p>Public or private harassment</p></li>
<li><p>Publishing others’ private information, such as a physical or email address,
without their explicit permission</p></li>
<li><p>Other conduct which could reasonably be considered inappropriate in a
professional setting</p></li>
</ul>
</section>
<section id="enforcement-responsibilities">
<h4>Enforcement Responsibilities<a class="headerlink" href="#enforcement-responsibilities" title="Link to this heading"></a></h4>
<p>Community leaders are responsible for clarifying and enforcing our standards of
acceptable behavior and will take appropriate and fair corrective action in
response to any behavior that they deem inappropriate, threatening, offensive,
or harmful.</p>
<p>Community leaders have the right and responsibility to remove, edit, or reject
comments, commits, code, wiki edits, issues, and other contributions that are
not aligned to this Code of Conduct, and will communicate reasons for moderation
decisions when appropriate.</p>
</section>
<section id="scope">
<h4>Scope<a class="headerlink" href="#scope" title="Link to this heading"></a></h4>
<p>This Code of Conduct applies within all community spaces, and also applies when
an individual is officially representing the community in public spaces.
Examples of representing our community include using an official e-mail address,
posting via an official social media account, or acting as an appointed
representative at an online or offline event.</p>
</section>
<section id="enforcement">
<h4>Enforcement<a class="headerlink" href="#enforcement" title="Link to this heading"></a></h4>
<p>Instances of abusive, harassing, or otherwise unacceptable behavior may be
reported to the community leaders listed
<a class="reference external" href="https://github.com/Slicer/Slicer/blob/master/GOVERNANCE.md#benevolent-dictators-for-life">here</a>
via private message on the <a class="reference external" href="https://discourse.slicer.org">Slicer forum</a>.
All complaints will be reviewed and investigated promptly and fairly.</p>
<p>All community leaders are obligated to respect the privacy and security of the
reporter of any incident.</p>
</section>
<section id="attribution">
<h4>Attribution<a class="headerlink" href="#attribution" title="Link to this heading"></a></h4>
<p>This Code of Conduct is adapted from the <a class="reference external" href="https://www.contributor-covenant.org">Contributor Covenant</a>,
version 2.1, available at
<a class="reference external" href="https://www.contributor-covenant.org/version/2/1/code_of_conduct.html">https://www.contributor-covenant.org/version/2/1/code_of_conduct.html</a>.</p>
<p>Community Impact Guidelines were inspired by
<a class="reference external" href="https://github.com/mozilla/diversity">Mozilla’s code of conduct enforcement ladder</a>.</p>
<p>For answers to common questions about this code of conduct, see the FAQ at
<a class="reference external" href="https://www.contributor-covenant.org/faq">https://www.contributor-covenant.org/faq</a>. Translations are available at
<a class="reference external" href="https://www.contributor-covenant.org/translations">https://www.contributor-covenant.org/translations</a>.</p>
</section>
<section id="deviations">
<h4>Deviations<a class="headerlink" href="#deviations" title="Link to this heading"></a></h4>
<p>For a history of updates and revisions to this document,
see <a class="reference external" href="https://github.com/Slicer/Slicer/commits/main/CODE_OF_CONDUCT.md">https://github.com/Slicer/Slicer/commits/main/CODE_OF_CONDUCT.md</a>.</p>
<!--
  Notes: Includes GOVERNANCE.md content, excluding the top-level header, intro,
  and the Code of Conduct section (already included above).
-->
</section>
</section>
<section id="decision-making-process">
<h3>Decision-making process<a class="headerlink" href="#decision-making-process" title="Link to this heading"></a></h3>
<ol class="arabic simple">
<li><p>Given the topic of interest, initiate discussion on the <a class="reference external" href="https://discourse.slicer.org">Slicer forum</a>.</p></li>
<li><p>Identify a small circle of community members that are interested to study the
topic in more depth.</p></li>
<li><p>Take the discussion off the general list, work on the analysis of options and
alternatives, summarize findings on the wiki or similar. <a class="reference external" href="https://www.slicer.org/wiki/Documentation/Labs">Labs</a>
page are usually a good ground for such summary.</p></li>
<li><p>Announce on the <a class="reference external" href="https://discourse.slicer.org">Slicer forum</a> the in-depth discussion of the topic for the
<a class="reference external" href="https://discourse.slicer.org/c/community/hangout">Slicer Community hangout</a>,
encourage anyone that is interested in weighing in on the topic to join the
discussion. If there is someone who is interested to participate in the discussion,
but cannot join the meeting due to conflict, they should notify the leaders of
the given project and identify the time suitable for everyone.</p></li>
<li><p>Hopefully, reach consensus at the hangout and proceed with the agreed plan.</p></li>
</ol>
<p><em>The initial version of these guidelines was established during the <a class="reference external" href="https://www.na-mic.org/Wiki/index.php/2017_Winter_Project_Week/UpdatingCommunityForums">winter
project week 2017</a>.</em></p>
</section>
<section id="benevolent-dictators-for-life">
<h3>Benevolent dictators for life<a class="headerlink" href="#benevolent-dictators-for-life" title="Link to this heading"></a></h3>
<p>The <a class="reference external" href="https://slicer.readthedocs.io/en/latest/user_guide/about.html#benevolent-dictators-for-life">benevolent dictators</a> can
integrate changes to keep the platform healthy and help interpret
or address conflict related to the contribution guidelines.</p>
<p>These currently include:</p>
<ul class="simple">
<li><p>Jean-Christophe Fillion-Robin</p></li>
<li><p>Andras Lasso</p></li>
<li><p>Steve Pieper</p></li>
</ul>
<p><em>Alphabetically ordered by last name.</em></p>
<p>The Slicer community is inclusive and welcomes anyone to work to become a core
developer and then a BDFL. This happens with hard work and approval of the existing
BDFL.</p>
</section>
</section>
<section id="contact-us">
<h2>Contact us<a class="headerlink" href="#contact-us" title="Link to this heading"></a></h2>
<p>It is recommended to post any questions, bug reports, or enhancement requests to the <a class="reference external" href="https://discourse.slicer.org">Slicer forum</a>.</p>
<p>Our online issue tracker is available <a class="reference external" href="https://issues.slicer.org">here</a>.</p>
<p>For commercial/confidential consulting, contact one of the <a class="reference internal" href="#commercial-partners">commercial partners</a>.</p>
</section>
</section>


           </div>
          </div>
          <footer><div class="rst-footer-buttons" role="navigation" aria-label="Footer">
        <a href="../index.html" class="btn btn-neutral float-left" title="Welcome to 3D Slicer’s documentation!" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
        <a href="getting_started.html" class="btn btn-neutral float-right" title="Getting Started" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right" aria-hidden="true"></span></a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>&#169; Copyright 2025, Slicer Community.</p>
  </div>

  Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    provided by <a href="https://readthedocs.org">Read the Docs</a>.
   

</footer>
        </div>
      </div>
    </section>
  </div>
  <script>
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script> 

</body>
</html>